conf/zeppelin-env.sh.template

```diff
@@ -26,6 +26,8 @@
 # export ZEPPELIN_LOG_DIR        # Where log files are stored.  PWD by default.
 # export ZEPPELIN_PID_DIR        # The pid files are stored. /tmp by default.
 # export ZEPPELIN_NOTEBOOK_DIR   # Where notebook saved
+# export ZEPPELIN_BUCKET         # Bucket where notebook saved
+# export ZEPPELIN_USER           # User in bucket where notebook saved. For example bucket/user/notebook/2A94M5J1Z/note.json
```

Comment:
Thanks for the contribution.
The name 'ZEPPELIN_BUCKET` and `ZEPPELIN_USER` little bit ambiguous to me.
How about having some prefix for S3NotebookRepo?

For example, `ZEPPELIN_NOTEBOOK_S3_BUCKET`, `ZEPPELIN_NOTEBOOK_S3_USER`, or any better name.